crates/curp/src/client/mod.rs

```diff
@@ -158,6 +160,8 @@ pub struct ClientBuilder {
     cluster_version: Option<u64>,
     /// initial cluster members
     all_members: Option<HashMap<ServerId, Vec<String>>>,
+    /// is current client send request to raw curp server
+    is_raw_curp: bool,
```

Comment:
what does this `is_raw_curp` means? It seems it's only be `true` in tests.

Reply:
`is_raw_curp` means that the client will send the request directly to the peer urls instead of the client urls, When `is_raw_curp` is set to true, the request does not go through AuthWrapper,